app/controllers/application_controller.rb

```diff
@@ -19,4 +20,11 @@ def redirect_to_setup
   def setup_done?
     !Minion.assigned_role.count.zero?
   end
+
+  # TODO: (mssola) remove once we have a reverse-proxy setup in place.
+  def redirect_to_secure
+    return if params[:action] == "autoyast" || params[:controller] == "health"
```

Comment:
It's weird to see in the application controller refer to the `action`, or even the `controller`... why not have a unconditional `before_action :redirect_to_secure` and on the dashboard controller: `skip_before_action :redirect_to_secure, only: :autoyast` and on the health controller `skip_before_action :redirect_to_secure` ?